### ISSUE TITLE
Gene name appearing as '_'

### ISSUE THREAD

**Issue by abeu1990** [tseemann/abricate#92]
Abricate is listing some genes as '_' for the gene name. It appears it may be the colistin resistance genes however other genes may also be impacted. See screen shot from abricate output.

![Abricate](https://user-images.githubusercontent.com/46658905/60489555-f72fe380-9ce7-11e9-9649-0153fa7da7f2.png)

**Comment by tseemann** [comment]
Thanks for submitting this bug @abeu1990 !
You are right about it being an _mcr_ gene too:

```
mcr-7.1_1_MG267386      dna     1620
mcr-9_1_NZ_NAAN01000063.1       dna     1620

resfinder~~~_~~~        dna     1620
resfinder~~~mcr-7.1_1~~~MG267386        dna     1620
```

**Comment by tseemann** [comment]
@abeu1990 i would recommend using `--db ncbi` in future. It is now the default database.

**Comment by tseemann** [comment]
@abeu1990 thank you so much for reporting this obscure bug!  I have fixed it in 0.9.4
